test/mpi/session/session_non_coll.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+
+#include "mpi.h"
+#include "stdio.h"
+
+int main(int argc, char *argv[])
+{
+    int ret;
+    MPI_Session session;
+    MPI_Group group;
+    int rank;
+
+    MPI_Session_init(MPI_INFO_NULL, MPI_ERRORS_ABORT, &session);
+    MPI_Group_from_session_pset(session, "mpi://world", &group);
+    MPI_Group_rank(group, &rank);
```

Comment:
Also check size and the tests require size > 1

Reply:
new version does this